### PR TITLE
Updated COMPONENT_VERSION file for release-2.8

### PR DESCRIPTION
**Pull request by dislbenn**
Signed-off-by: dislbenn <dbennett@redhat.com>

### CODE REVIEW

File: COMPONENT_NAME [hunk 1]
```diff
@@ -0,0 +1 @@
+kube-rbac-proxy
```
Comment (subbarao-meduri) [inline COMPONENT_NAME]: Should the name be `prometheus`?

**Review by subbarao-meduri — APPROVED** [review]
/lgtm

File: COMPONENT_NAME (hunk 1 above)
Comment (dislbenn) [reply COMPONENT_NAME]: Yes, this has been updated.

**Pull request merged**